fixtures/u2_import_path.rb.sem.uast

```diff
@@ -0,0 +1,50 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "begin",
+      '@role': [Block],
+      '@pos': { '@type': "uast:Positions",
+         start: { '@type': "uast:Position",
+            offset: 0,
+            line: 1,
+            col: 1,
+         },
+         end: { '@type': "uast:Position",
+            offset: 57,
+            line: 2,
+            col: 32,
+         },
+      },
+      body: [
+         { '@type': "uast:QualifiedIdentifier",
```

Comment:
An import transform seems to be broken. It should return an `Import` or `RuntimeImport` with a string argument (path)

Reply:
Yeah, this seem to be wrong in (our layer over) the native AST, it should have the type "require" but has the type "send_qualified", will check it and write the canonical "done" here when fixed.

Reply:
done